plutus-tx-plugin/src/PlutusTx/Plugin.hs

```diff
@@ -161,6 +161,7 @@ mkSimplPass flags logger =
             , GHC.sm_uf_opts = GHC.defaultUnfoldingOpts
             , GHC.sm_dflags = flags
             , GHC.sm_rules = False
+            , GHC.sm_builtin_rules = False
```

Comment:
Nice! Add a comment here or a note on why we have this set to False (default for ghc being True)